SDL/Quintuplet.h

```diff
@@ -3150,52 +3148,7 @@ namespace SDL {
 
         int nEligibleT5Modules = alpaka::atomicOp<alpaka::AtomicAdd>(acc, &nEligibleT5Modulesx, 1);
 
-        if (module_layers <= 3 && module_subdets == 5)
-          category_number = 0;
-        else if (module_layers >= 4 && module_subdets == 5)
-          category_number = 1;
-        else if (module_layers <= 2 && module_subdets == 4 && module_rings >= 11)
-          category_number = 2;
-        else if (module_layers >= 3 && module_subdets == 4 && module_rings >= 8)
-          category_number = 2;
-        else if (module_layers <= 2 && module_subdets == 4 && module_rings <= 10)
-          category_number = 3;
-        else if (module_layers >= 3 && module_subdets == 4 && module_rings <= 7)
-          category_number = 3;
-        else
-          category_number = -1;
-
-        if (module_eta < 0.75)
-          eta_number = 0;
-        else if (module_eta > 0.75 && module_eta < 1.5)
-          eta_number = 1;
-        else if (module_eta > 1.5 && module_eta < 2.25)
-          eta_number = 2;
-        else if (module_eta > 2.25 && module_eta < 3)
-          eta_number = 3;
-        else
-          eta_number = -1;
-
-        if (category_number == 0 && eta_number == 0)
-          occupancy = 336;
-        else if (category_number == 0 && eta_number == 1)
-          occupancy = 414;
-        else if (category_number == 0 && eta_number == 2)
-          occupancy = 231;
-        else if (category_number == 0 && eta_number == 3)
-          occupancy = 146;
-        else if (category_number == 3 && eta_number == 1)
-          occupancy = 0;
-        else if (category_number == 3 && eta_number == 2)
-          occupancy = 191;
-        else if (category_number == 3 && eta_number == 3)
-          occupancy = 106;
-        else {
-          occupancy = 0;
-#ifdef Warnings
-          printf("Unhandled case in createEligibleModulesListForQuintupletsGPU! Module index = %i\n", i);
-#endif
-        }
+        occupancy = 2 * tripletsInGPU.nTriplets[i];
```

Comment:
this will look bad; probably more so at high pt.
But we need breakdown plots; CI is not showing them

Reply:
Here are the breakdown plots, I don't see any issues:

https://www.classe.cornell.edu/~gsn27/www/www/PR406/

Reply:
ehm, I was wrong at least at base/integrated  level  ;)

Reply:
> You can also see the breakdown plots from the CI by downloading the full zip file.

ah OK.



> Here are the breakdown plots. I don't see any issues, maybe I'm missing something though.

yes, it looks OK; it takes an effort to see a difference.

The other side (opposite to high pt high occupancy) to worry about is the low end: a module with one triplet can have more than two matching. This may become more visible without PU.

Reply:
the low occupancy end could be moderated by e.g. `2*n_T3 + 10` ("random" guess for values; I'm still implying some plots need to be made to show the boundary is OK).